### ISSUE TITLE
[Suggestion] Apply changes to Fog aswell

### ISSUE THREAD

**Issue by DerCommander323** [someaddons/betterfpsdistances#21]
Since this mod changes only the actual render distance, it causes the fog to not be visible when the ground you're viewing is on a different height level than the camera. My suggestion is to just apply the same shape as the render distance to the fog aswell (if that is possible), to make the fog always match.

**Comment by Nekkowe** [comment]
Yeah, with the fog not matching up, regular common sights like e.g. a forest seen from atop a mountain honestly look atrocious

**Comment by someaddons** [comment]
difficult since the fog is shader based

**Comment by Nekkowe** [comment]
I'd happily install the shader alongside this, if that's what it takes

**Comment by mateus3c** [comment]
yeah, the fog looks really ugly with betterfps but maybe using "fog looks good now" fixes that?
